imap_processing/tests/hit/test_housekeeping.py

```diff
@@ -0,0 +1,92 @@
+import pathlib
+
+import numpy as np
+import pandas as pd
+
+from imap_processing import decom
+from imap_processing.hit.l0.data_classes.housekeeping import Housekeeping
+
+
+def test_houskeeping():
+    # The HK validation data's first row was not in the CCSDS file sent, so that has
+    # been manually removed from the validation file on the SDC end. The last
+    # packet in the CCSDS file also does not correspond to any of the rows in
+    # the validation file, so the last packet in the CCSDS file is removed for
+    # this test. These issues were discovered / confirmed with the HIT Ops
+    # engineer who delivered the data.
```

Comment:
What joy :) Thanks for your patience in finding these! And especially documenting this.